tools/stress-cluster/cluster/kubernetes/stress-test-addons/images/test-resource-deployer/prepare.ps1

```diff
@@ -0,0 +1,4 @@
+$copyDirectory = Resolve-Path (Join-Path -Path $PSScriptRoot -ChildPath "../../../../../../../eng/common/")
```

Comment:
`"../../../../../../../eng/common/"` goodness...

Perhaps we want to find the git root here instead?